src/pages/new/index.js

```diff
@@ -23,6 +23,8 @@ import image2 from "../../components/grid-aware/ThreeParagraphBlock/stories/imag
 import image3 from "../../components/grid-aware/ThreeParagraphBlock/stories/image3.png";
 import VideoHeader from "../../components/grid-aware/VideoHeader/VideoHeader";
 import videoHeaderImage from "../../components/grid-aware/VideoHeader/stories/VideoHeader.png";
+import VideoSpotlightBlock from "../../components/grid-aware/VideoSpotlightBlock";
+import videoSpotlightBlockImage from "../../components/grid-aware/VideoSpotlightBlock/stories/VideoSpotlightBlock.svg";
```

Comment:
Could you use a PNG and not an SVG for this image? This ended up embedding a PNG into the SVG anyway, since SVGs can't be used to encode non-vector data, like an actual photograph.

Reply:
ok